### PR TITLE
The Extremely Rad Everything Tengoku Pack

### PR DESCRIPTION
**Pull request by GuardedLolz**
### Prerequisites
* [X] I have read and understood the [contribution guidelines](CONTRIBUTING.md)
* [X] This pull request follows the code style of the project
* [x] I have tested this feature thoroughly WITH THE USE of the load-all-sounds starter
  * [Link to a video of proof of testing]()

### Changes Proposed in this Pull Request
**BIG ROCK FINISH**
Add fast cues
**BLUE BIRDS**
Add Japanese and Spanish SFX.
**BON ODORI**
Completely revamped. (thankfully)
**FIREWORKS**
Is now segmented and the fireworks get their real names. The Rhythm Rockets are Base BPM at the moment, because the music decided to interfere.
**HEAD SPINNER**
The Bracelety of Rhythm Heaven finally comes to the Remix Editor.
**NIGHT WALK**
Added wot and different jump layouts.
**POLYRHYTHM**
Added clear sound.
**POWER CALIGRAPHY**
Added. Though I wonder why it wasn't here in the first place.
**QUIZ SHOW**
Added a version of the correct answer sound without the fanfare.
**RAP MEN**
Added and fixed patterns. It is now officially bigger than Ringside.
**SPACE DANCE**
Added Cosmic Dance.
**TAP TRIAL**
Added Remix 7 version. Icon could be better.
**TOSS BOYS**
Segmented super start sounds for Ao-kun and Aka-chan. Added Aka-chan's unused super.

...

### CODE REVIEW

**Comment by chrislo27** [comment]
You didn't change games.json. Do not check off the "I have tested this feature thoroughly" checkbox until you have done so!

**Comment by chrislo27** [comment]
Is it really that hard to test? `fanClubEn` doesn't exist (there's a `fanClub` folder though, no deprecated IDs though).

**Comment by chrislo27** [comment]
Also, do you have a Discord account? Contacting me in (more) real time is easier on the Rhythm Heaven server (we have a rhre2 channel). The invite link is in the sidebar of https://www.reddit.com/r/rhythmheaven/.

**Comment by chrislo27** [comment]
Deprecations missing:
```
bonOdori/dondon1, bonOdori/dondon2, bonOdori/pan, bonOdori/panpan1, bonOdori/panpan2, nightWalk_jump, tossBoys/superRed, tossBoys/superBlue
```

File: core/assets/data/games.json [hunk 1]
```diff
@@ -40,25 +42,27 @@
...
   "glassTappers",
   "gleeClub",
+  "headSpinner",
   "holeInOne",
   "karateMan",
   "karateManGba",
   "karateManDS",
   "kitties",
   "launchParty",
+  "lockstep64",
```
Comment (chrislo27) [inline core/assets/data/games.json]: Please remove (converted to a module for v2.10.2).

File: core/assets/data/games.json [hunk 2]
```diff
@@ -97,10 +102,12 @@
   "tambourine",
   "tangotronic3000",
   "tapTrial",
+  "tapTrialRemix7",
   "tapTroupe",
   "theDazzles",
   "tongueLashing",
   "tossBoys",
+  "tramAndPauline",
```
Comment (chrislo27) [inline core/assets/data/games.json]: Please remove, (data.json) doesn't exist yet.

**Review by chrislo27 — CHANGES_REQUESTED** [review]

**Comment by chrislo27** [comment]
Overall, I am incredibly happy with this pull request. It's clear you put a lot of time and effort into this, and it shows! :+1:

File: core/assets/sounds/cues/rapMen/data.json [hunk 3]
```diff
@@ -45,63 +80,118 @@
...
     {
-      "id": "rapMen/saiko2",
+      "id": "rapMen/orenosei3",
       "duration": 0.375,
-      "name": "saiko - ko!"
+      "name": "orenosei - no"
     },
     {
-      "id": "rapMen/oishii1",
+      "id": "rapMen/orenosei1",
```
Comment (chrislo27) [inline core/assets/sounds/cues/rapMen/data.json]: Accidental duplicate `orenosei` cues? (1, 2, 3)

**Review by chrislo27 — CHANGES_REQUESTED** [review]

**Comment by chrislo27** [comment]
Also, I think Cosmic Dance needs a swung and pose/turn right cue. Not too sure though.

**Comment by GuardedLolz** [comment]
I think Cosmic Dance poses are just the original Space Dance pose, but on the offbeat. Also, I said the icon for Remix 7 Tap Trial is a placeholder. Can you update it to fit in with the rest of the games?

File: core/assets/sounds/cues/rapMen/data.json (hunk 3 above)
Comment (GuardedLolz) [reply core/assets/sounds/cues/rapMen/data.json]: Fixed.

File: core/assets/data/games.json (hunk 1 above)
Comment (GuardedLolz) [reply core/assets/data/games.json]: What's a module?

**Comment by chrislo27** [comment]
I think the "turn" part is shifted forward by an sixth of a beat (like your other cosmic patterns). Better to have it for consistency than not at all, I suppose.

**Comment by GuardedLolz** [comment]
Apart from being on the offbeat, the cosmic pose cue has nothing that sets it apart from the normal cue.

**Comment by chrislo27** [comment]
Fair enough. Please fix the rest of the things I pointed out in the review and I might try to get a snapshot out today for testing.

**Comment by GuardedLolz** [comment]
I did.

**Comment by chrislo27** [comment]
Bleh, GH only *sometimes* updates commits in real-time. I'll check them now, sorry!

**Comment by GuardedLolz** [comment]
I just forgot to sync my commit,

**Comment by GuardedLolz** [comment]
The very next thing I am going to do after this is add more foreign sounds.

**Pull request merged**